northstar/src/npk/manifest.rs

```diff
@@ -150,6 +152,20 @@ impl Manifest {
                 _ => Ok(()),
             })?;
 
+        // Check selinux context type
+        if let Some(selinux) = &self.selinux {
```

Comment:
Is there a length restriction on the labels?

Reply:
I think we can use
```
#define XATTR_SIZE_MAX 65536
```
(https://www.spinics.net/lists/selinux/msg09373.html)